src/environments/environment.nonprod.ts

```diff
@@ -0,0 +1,31 @@
+// This file can be replaced during build by using the `fileReplacements` array.
+// `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
+// The list of file replacements can be found in `angular.json`.
+
+export const environment = {
+  get production() {
+    return false;
+  },
+  get name() {
+    return 'nonproduction';
+  },
+  get apis() {
+    return {
+      get questionServiceConsumerEndpoint() {
+        return 'https://question-service-ft1.azurewebsites.net/api/question';
```

Comment:
For reference (we an change it some other time), we'd generally make non-prod = develop (rather than a branch that will be deleted at some point), preprod = master and master as a tagged release on Github (maybe 1.1.0 or something). Reason being you don't want any code that goes into master to immediately get to customers without a formal release process.